docs/source/examples/cirq-ibmq-backends.myst

```diff
@@ -0,0 +1,264 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(label-cirq-zne-example)=
+# Error mitigation on IBMQ backends with Cirq
+
+
+This tutorial shows an example of how to mitigate noise on IBMQ
+backends with the Cirq frontend, broken down in the following steps.
+
+- [Error mitigation on IBMQ backends with Cirq](#error-mitigation-cirq-on-ibmq-backends)
+  - [Settings](#settings)
+  - [Setup: Defining a circuit](#setup-defining-a-circuit)
+  - [High-level usage](#high-level-usage)
+  - [Options](#options)
+  - [Lower-level usage](#lower-level-usage)
+
++++
+
+## Settings
+
+```{code-cell} ipython3
+from mitiq import zne
+from mitiq.interface.mitiq_qiskit.qiskit_utils import initialized_depolarizing_noise
+```
+
+**Note:** If `USE_REAL_HARDWARE` is set to `False`, a classically simulated noisy backend is used instead of a real quantum computer.
+
+```{code-cell} ipython3
+USE_REAL_HARDWARE = False
+```
+
+(examples/cirq-ibmq-backends/setup-defining-a-circuit)=
+## Setup: Defining a circuit
+
++++
+
+For simplicity, we'll use a random single-qubit circuit with ten gates that compiles to the identity, defined below.
+
+(examples/cirq-ibmq-backends/cirq-frontend)=
+### Cirq frontend
```

Comment:
Unnecessary. If we wanted to emphasize cirq here, maybe rename the header above to `Setup: defining a circuit with cirq`.

Reply:
Agree, I've consolidated the sections and headings accordingly.